cloverage/test/cloverage/coverage_test.clj

```diff
@@ -197,15 +196,15 @@
 (t/deftest test-instrument-gets-lines
   (inst/instrument #'cov/track-coverage
                    'cloverage.sample.exercise-instrumentation)
-  (let [cov @cov/*covered*
-        found (find-form cov '(+ 40 2))]
-    #_(with-out-writer "out/foo"
-        (doseq [form-info cov]
-          (println form-info)))
-    #_(println "Form is")
-    (t/is found)
-    (t/is (:line found))
-    (t/is (find-form cov '(inc (m c 0))))))
+  (let [cov @cov/*covered*]
+    (doseq [[form expanded] '{(+ 40)      (+ 40)
+                              (+ 40 2)    (. clojure.lang.Numbers (add 40 2))
+                              (str 1 2 3) (str 1 2 3)
+                              (inc m c 0) (. clojure.lang.Numbers (inc (m c 0)))}]
+      (t/testing (format "Form %s (expanded to %s) should get instrumented" (pr-str form) (pr-str expanded))
+        (t/is (find-form cov expanded))
+        (let [found (find-form cov expanded)]
+          (t/is (:line found)))))))
```

Comment:
Reworked this test to use `doseq` so we can include `testing` context info in there to make it easier to distinguish failures for the various forms. I kept the original two forms tested here (both of which are now correctly expanded to inlined versions) and added a couple more that aren't expanded